src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Strings.Designer.cs

```diff
@@ -8,34 +8,35 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-namespace NuGet.Build.Tasks.Pack {
+namespace NuGet.Build.Tasks.Pack.Library {
```

Comment:
All other classes are in `NuGet.Build.Tasks.Pack` namespace. Any reason for this change? It pollutes usings with inobvious  entries :(